ml-agents/mlagents/trainers/tests/test_trainer_controller.py

```diff
@@ -367,29 +367,6 @@ def trainer_controller_with_take_step_mocks():
 
     return tc, trainer_mock
 
-
-def test_take_step_resets_env_on_global_done():
```

Comment:
I'd rather not remove unit tests. If you could convert it to test for the subprocess_environment, that would be ideal.
Can you test that this works with curriculum ?

Reply:
Agreed on the test.

🤔 this didn't touch any code interacting with curriculum -- unfortunately since we don't have any recurring curriculum tests it will be hard to be sure what effect it had but I will try to run an example.

Reply:
Added a new test and tested with curriculum -- everything seems 👌